src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.cs

```diff
@@ -1405,26 +1586,33 @@ private static Args ParseFromCommandLine(string[] args)
         uint pohAllocLow = DEFAULT_POH_ALLOC_LOW;
         uint pohAllocHigh = DEFAULT_POH_ALLOC_HIGH;
 
+        uint reqSohSurvInterval = DEFAULT_REQ_SOH_SURV_INTERVAL;
+        uint reqLohSurvInterval = DEFAULT_REQ_LOH_SURV_INTERVAL;
 #if NET5_0_OR_GREATER
         uint pohFinalizableInterval = DEFAULT_POH_FINALIZABLE_INTERVAL;
         uint pohSurvInterval = DEFAULT_POH_SURV_INTERVAL;
+        uint reqPohSurvInterval = DEFAULT_REQ_POH_SURV_INTERVAL;
 #endif
         uint pohAllocRatioArg = 0;
 
         ItemType allocType = ItemType.ReferenceItem;
         bool verifyLiveSize = false;
         uint printEveryNthIter = 0;
         bool finishWithFullCollect = false;
-        bool compute = false;
+        uint compute = 0;
         bool endException = false;
 
+        ulong requestAllocBytes = 0;
+        ulong requestLiveBytes = 0;
+        uint sizeDist = 0;
+
         for (uint i = 0; i < args.Length; ++i)
         {
             switch (args[i])
             {
                 case "-compute":
                 case "-c":
-                    compute = true;
+                    compute = 1000;
```

Comment:
Might be worth adding a line or two about why this is value is hardcoded and the significance of the value?

Reply:
Agreed - done.